packages/suite/src/hooks/wallet/coinmarket/form/common/useCoinmarketFormActions.ts

```diff
@@ -203,8 +177,7 @@ export const useCoinmarketFormActions = <T extends CoinmarketSellExchangeFormPro
             // set token address for ERC20 transaction to estimate the fees more precisely
```

Comment:
please again, use `networkType === 'ethereum'` 
btw this place smells like a bug ❗ you do it only for ethereum, not for polygon, bnb,...